src/index.html

```diff
@@ -3,11 +3,287 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@700&family=Space+Grotesk:wght@700&family=Space+Mono&display=swap" rel="stylesheet">
+    <title>B&O</title>
+    <link rel="stylesheet" href="./main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header" id="home">
+      <navigation class="navigation header__navigation">
+        <a class="logo" href="#">
+          <img class="logo__icon"
+          src="images/Logo.svg" alt="logo">
+        </a>
+
+        <div class="navigation__buttons">
+
+          <a class="phone" href="tel:+1 234 555-55-55">
+            <span class="phone__label">tel:+1 234 555-55-55</span>
+            <img
+              class="phone__icon"
+              src="images/Icon-Phone-call.svg"
+              alt="icon-phone">
+          </a>
+          <a class="burger" href="#menu"><span class="burger__label">Open menu</span></a>
+        </div>
+      </navigation>
+      <h1 class="title header__title">
+        Bring joy back to the everyday
+      </h1>
+    </header>
+
+    <div class="menu" id="menu">
+      <navigation class="navigation">
+        <a class="logo" href="#">
+          <img class="logo__icon"
+          src="images/Logo.svg" alt="logo">
+        </a>
+
+        <div class="navigation__buttons">
+          <a
+            class="close"
+            href="#home"
+          >
+            <img
+              class="close__icon"
+              src="images/Icon-Close.svg"
+              alt="close">
+          </a>
+        </div>
+      </navigation>
+
+      <navigation class="links menu__links">
+        <ul class="links__list">
+          <li class="links__item"><a class="links__link" href="#products">Products</a></li>
```

Comment:
Please format code to be more readable: 

```suggestion
          <li class="links__item">
            <a class="links__link" href="#products">Products</a>
          </li>
```